app/controllers/api/services_controller.rb

```diff
@@ -30,8 +30,12 @@ def edit
   end
 
   def settings
-    activate_menu_args = current_account.provider_can_use?(:api_as_product) ? %i[serviceadmin applications usage_rules] : %i[serviceadmin integration settings]
-    activate_menu activate_menu_args
+    activate_menu :serviceadmin, :integration, :settings
+    @alert_limits = Alert::ALERT_LEVELS
+  end
+
+  def usage_rules
```

Comment:
It is under rolling update right?

Reply:
The link is under rolling updates, here: https://github.com/3scale/porta/blob/THREESCALE-3471/enhance_for_3451/app/helpers/vertical_nav_helper.rb#L224-L227